src/lightkurve/lightcurve.py

```diff
@@ -3140,7 +3204,141 @@ def plot_river(self, **kwargs):
+        time_bin_start : Time like value, optional
+            The start phase for the binned time series. This can also be a scalar
+            value if ``time_bin_size`` is provided. Defaults to the first
+            time in the sampled time series.
+        aggregate_func : callable, optional
+            The function to use for combining points in the same bin. Defaults
+            to np.nanmean.
+        bins : int, optional
+            int which gives the number of bins to divide the lightkurve into.
+            This adjusts the length of ``time_bin_size``
+            to accommodate the input time series length.
+        n_bins: 
+            This functionality is deprecated for FoldedLightCurves. If provided, asserts
+            bins=n_bins
+
+        Returns
+        -------
+        binned_lc : `FoldedLightCurve`
+            A new folded light curve which has been binned, with the 'time' column phase in days 
+        """
+        # astropy's aggregate_downsample function only works when 'time' is type Time or TimeDelta
+        # Since the normalized phase is a unitless quantity, this breaks.
+        # To work around this, we reset the index to be the regular phase (TimeDelta) when binning
+        
+        if n_bins != None:
+            warnings.warn("n_bins is no longer accepted for FoldedLightCurve objects. Please specify 'bins' instead")
+            bins = n_bins
+        if bins != None:
+            if not isinstance(bins, int):
+                raise ValueError('bins must be an integer describing the total number of bins.')
+            if time_bin_size != None:
+                raise ValueError("Can not specify both 'bins' and 'time_bin_size'")
+            
+
+        if self.normalize_phase == True:
+            self._replace_normalized_phase()
+
+        if time_bin_start is None:
+            time_bin_start = self.time[0]
+        if isinstance(time_bin_start, list):
+            time_bin_start = time_bin_start[0]
+            warnings.warn("FoldedLightCurve does not support lists for time_bin_start. Using the first value.")
+        if isinstance(time_bin_start, Quantity):
+            if time_bin_start.unit == u.dimensionless_unscaled:
+                    time_bin_start = time_bin_start.value * self.period      
+        if np.isscalar(time_bin_start):
+            time_bin_start = time_bin_start * u.day
+
+
+
+        if time_bin_size != None:
+            if np.isscalar(time_bin_size):
+                if self.normalize_phase == True:
+                    time_bin_size = time_bin_size * u.dimensionless_unscaled
+                else:
+                    time_bin_size = time_bin_size * u.day
+            if isinstance(time_bin_size, Quantity):
+                if time_bin_size.unit == u.dimensionless_unscaled:
+                    time_bin_size = time_bin_size.value * self.period
+ 
+
+        if bins != None:
+            # astropy's aggregate_downsample doesn't work for phase data:
```

Comment:
I just realized the comment `astropy's aggregate_downsample doesn't work for phase data` is incorrect.  The problem of `bin(bins=###)` is due to lightkurve's own `Lighcurve.bin()` codes that assume time column is of `Time` type.

https://github.com/lightkurve/lightkurve/blob/b81ebc207a5cad3040659fd141f75fa7136a678f/src/lightkurve/lightcurve.py#L1543-L1544

It'd be better to reword it to something like "parent bin() does not work for phase data"